db_test.go

```diff
@@ -27,12 +28,10 @@ func TestGinkgo(t *testing.T) {
 
 func pgOptions() *pg.Options {
 	return &pg.Options{
-		User:     "postgres",
-		Database: "postgres",
+		User:     pg.GetPGUser(),
+		Database: pg.GetPGDatabase(),
```

Comment:
I think you can leave these empty so they are auto-populated later in options.go

Reply:
Thanks, done.